### ISSUE TITLE
Namadillo: Progress bars and confetti for transaction status and shielded-sync

### ISSUE THREAD

**Issue by Rigorously** [anoma/namada-interface#1418]
Whenever starting a transaction, the interface is more or less static, except for maybe a faintly blinking yellow circle. In fact, it takes a few seconds before the Keychain asks the user to sign the transaction. 

![image](https://github.com/user-attachments/assets/0d9dd7e8-3ec0-4c4c-886b-203fdd59050d)

Then, when shielding, it takes a while for the ShieldedSyncWorker to catch up.  

![image](https://github.com/user-attachments/assets/927a6846-e3b7-44ec-b3a9-65a9999cbfa9)

And when the transaction is finished, the only indication is that the "Transfer Complete" becomes brighter. Very anticlimactic. I was still waiting for the transaction to finish up.

To the end-user, this dead time and too subtle visualization looks as if nothing is happening. I suggest to spice things up with progress bars and confetti. 🎉

Namadillo 1.0.6

**Comment by Rigorously** [comment]
I see there is a progress circle for shielded sync, but it has no label

![image](https://github.com/user-attachments/assets/d2714bcf-88a6-4f4f-a8c4-9df828b7a170)

**Comment by euharrison** [comment]
@ChrisHoltDesign do you have a suggestion here?

**Comment by ChrisHoltDesign** [comment]
Yes i was speaking to Pedro about this a while back to emphasise each state more as it's not noticeable enough. (Also feels like it stops at the wrong point)

I think a few things can be done to build on Rigourously is pointing out.

- Contrast between the active and non active / upcoming
- The whole thing is a progress bar but maybe we need a % amount at each stage so we know how it's progressing. Or just a whole percentage value at the bottom
- Confetti idea. I think maybe just a big check mark, something similar we have in the extension once keys are created. Can always make it vibrate and pulse with concentric rings (I can mock this up )

-----------
Shielded sync, worth having this in the Top bar next to the green dot? It can disappear once synched

**Comment by ChrisHoltDesign** [comment]
Ok Transaction complete animation

1. Screen flips to the asset / or assets that have been transferred to be in the center. (Concentric rings pulsate)

<img width="730" alt="Screenshot 2024-12-19 at 12 30 29" src="https://github.com/user-attachments/assets/1e3660fc-e8d6-4a69-8f3f-abbcf5d6a187" />

2. Asset logo then flips to yellow to become 'private' (Animaton then folds into the sent like the asset is sent into the page (MASP)

<img width="727" alt="Screenshot 2024-12-19 at 12 30 35" src="https://github.com/user-attachments/assets/d60ff0ca-231e-4c67-a1ff-fcca4f5aea2d" />

<img width="728" alt="Screenshot 2024-12-19 at 12 33 17" src="https://github.com/user-attachments/assets/b6ec5a66-a101-44b5-af73-e62bee33d279" />


For non shielded version the asset can just get sucked into the page , doesn't need to convert.

Can all happen in 2/3 seconds.

CC @pedrorezende

**Comment by ChrisHoltDesign** [comment]
For each stage if we increase the animation. Maybe a little movement pulse like we have on the namada logo here https://namada.net/mainnet-launch

**Comment by ChrisHoltDesign** [comment]
Add a little spinng 'shielded sync' icon next to the green dot. 

Operates in the same way as the green dot with a rollover to display info on perentage.

<img width="796" alt="Screenshot 2024-12-19 at 13 04 59" src="https://github.com/user-attachments/assets/edc851ce-1086-4464-93d7-f424d20b4762" />

For the display on the MASP page lets just add in a text signifier under the percentage.

On MASP and IBC we can display it in the transfer module if we are not allowing user actions during syncing
